pkg/helm/chartproxy/proxy_test.go

```diff
@@ -16,16 +17,29 @@ import (
 
 func TestProxy_IndexFile(t *testing.T) {
 	tests := []struct {
-		name       string
-		indexFiles []string
-		mergedFile string
-		helmCRS    []*unstructured.Unstructured
+		name           string
+		indexFiles     []string
+		mergedFile     string
+		kubeVersion    *chartutil.KubeVersion
```

Comment:
could we use string here, and tests values according to the doc syntax: https://helm.sh/docs/topics/charts/#the-kubeversion-field ?